### DISCUSSION TITLE
First Time Loading Study from Cache or Server

### DISCUSSION THREAD

**Discussion by Avinash-c-gowda** [nroduit/Weasis#499] [General]
Hi @nroduit, during the initialization of the framework, you are storing the study to cache. After that, when starting the framework, you load the study from  cache or server?   could u please clarify this.

Thanks in advance

**Comment by nroduit** [comment]
As there are several ways to load data and several retrieval protocols (C-GET, C-MOVE, WADO-RS, WADO-URI) that work differently, it is difficult to answer. Please specify the question.

**Reply by Avinash-c-gowda** [reply]
Hello @nroduit, I am inquiring about WADO-RS. I have a PACS server, and I launch my viewer from the URL of that server. What I observed during the initialization of the framework is that only the bundle is stored in the cache with the name cache-<weasisSourceID>.

However, my question is whether DICOM images are loaded for the first time from the cache or the PACS server when starting Weasis. Additionally, if I close Weasis, will the cache be cleared?

**Comment by nroduit** [comment]
Yes, remote images are downloaded locally to the system tmp folder and released when the application is closed or from the context menu of DICOM Explorer: remove patient/study/series.

**Reply by Avinash-c-gowda** [reply]
Hello, @nroduit. My idea is, I want to store a list of studies in the system temp folder. Is that a good approach?
if yes, 
If the studies are in the system temp folder, is the viewer checking the system temp folder and loading studies?
